### PR TITLE
SignIn SignUp design

### PR DESCRIPTION
**Pull request by anndanielyan**

### CODE REVIEW

File: style/style.css [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+body {
+  font-family: sans-serif;
+  color: #333;
+  line-height: 1.5;
+  height: 100vh;
+  background: linear-gradient(to right, #353a5f, #cacfe3);
+}
+
+h1 {
```
Comment (ghost) [inline style/style.css]: @anndanielyan add class name, and then give a styles

File: style/style.css [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  cursor: pointer;
+}
+
+.btn:hover {
+  background: linear-gradient(to left, #353a5f, #cacfe3);
+  color: #cacfe3;
+}
+
+@media (max-width: 42rem) {
+  h1 {
```
Comment (ghost) [inline style/style.css]: @anndanielyan same for here

File: style/style.css [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    font-size: 2.5rem;
+  }
+  .btn {
+    width: 25rem;
+    font-size: 1.6rem;
+  }
+}
+
+@media (max-width: 26.2rem) {
+  h1 {
```
Comment (ghost) [inline style/style.css]: @anndanielyan same for here

File: style/style.css [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+    font-size: 3rem;
+  }
+  .btn {
+    width: 30rem;
+    font-size: 1.7rem;
+  }
+}
+
+@media (max-width: 32rem) {
+  h1 {
```
Comment (ghost) [inline style/style.css]: @anndanielyan same for here

File: style/style.css [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+    font-size: 2rem;
+  }
+  .btn {
+    width: 20rem;
+    font-size: 1.5rem;
+  }
+}
+
+@media (max-width: 22rem) {
+  h1 {
```
Comment (ghost) [inline style/style.css]: @anndanielyan same for here

File: style/style.css [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+    font-size: 1.7rem;
+  }
+  .btn {
+    width: 15rem;
+    font-size: 1.4rem;
+  }
+}
+
+@media (max-width: 18rem) {
+  h1 {
```
Comment (ghost) [inline style/style.css]: @anndanielyan same for here

**Review by ghost — CHANGES_REQUESTED** [review]
@anndanielyan , you need to make changes, also, every comment in github and trello you must answer, if there is everything okay, I'll resole conversation, in other way, you need to make changes! One more time, you must answer each of comments, there will not be unanswered comment!!1

File: style/style.css (hunk 1 above)
Comment (anndanielyan) [reply style/style.css]: I know that every page must have only one h1. That's why I didn't add class, but now I will add it.

Comment (ghost) [reply style/style.css]: If one day we will change that h1 tag to h2, you need to change it everywhere, but if you add class, its doesn't mean which tag you will change

**Review by ghost — CHANGES_REQUESTED** [review]
@anndanielyan please remove bgc color, and change hover colors, it not looks good for eyes

File: style/style.css (hunk 1 above)
Comment (anndanielyan) [reply style/style.css]: ah yes, I understood. Will I open a new branch for this?

**Comment by anndanielyan** [comment]
> @anndanielyan please remove bgc color, and change hover colors, it not looks good for eyes
> @epamGevorg  Must I remove bgc at all? or change?

**Comment by ghost** [comment]
> Must I remove bgc at all? or change

If you're not sure that you can pick correct color, remove it, also don't forget change font styles, and change hover colors

File: style/style.css (hunk 1 above)
Comment (ghost) [reply style/style.css]: nope
just commit changes

**Comment by anndanielyan** [comment]
> > Must I remove bgc at all? or change
> 
> If you're not sure that you can pick correct color, remove it, also don't forget change font styles, and change hover colors

> I did it, but I am not sure that it is correct.

**Comment by ghost** [comment]
> > > Must I remove bgc at all? or change
> > 
> > 
> > If you're not sure that you can pick correct color, remove it, also don't forget change font styles, and change hover colors
> 
> > I did it, but I am not sure that it is correct.

Thats why we need PRs, push your changes, I'll check it one more time

**Comment by anndanielyan** [comment]
> > > > Must I remove bgc at all? or change
> > > 
> > > 
> > > If you're not sure that you can pick correct color, remove it, also don't forget change font styles, and change hover colors
> > 
> > 
> > > I did it, but I am not sure that it is correct.
> 
> Thats why we need PRs, push your changes, I'll check it one more time

> I pushed, but there is 2 commit, pls check second one )) There is comment in first one.

**Review by ghost — APPROVED** [review]
Looks like good

**Pull request merged**